### PR TITLE
(Un)Pause operations

### PR DESCRIPTION
**Pull request by cscetbon**
- Add 2 new options to pause/unpause operations
- Apply fields and labels selectors when looking for pods
- Update all labels of a pod at once

See https://pastebin.com/cZGPrpq3 for test cases

### CODE REVIEW

File: plugins/kubectl-casskop [hunk 1]
```diff
@@ -20,6 +20,11 @@ import json
 from subprocess import check_output, STDOUT, CalledProcessError
 from os.path import basename
 from random import shuffle
+from itertools import groupby
+
+PAUSE_LABELS_OP = ["ToDo", "Paused"]
+NO_PODS_FOUND = "No pods found for operation"
+NO_ONGOING_OP = "--selector=operation-status notin (Ongoing, Finalizing)"
```
Comment (allamand) [inline plugins/kubectl-casskop]: not in

Comment (cscetbon) [reply plugins/kubectl-casskop]: It’s not python here ^ but a selector filter expression.
The syntax is `notin` and not `not in` 🤷‍♂️

**Comment by cscetbon** [comment]
> maybe this is out of scope of this PR but seams to be addressed

Agreed. Let’s create an issue to add tests using the plugin.

File: plugins/kubectl-casskop (hunk 1 above)
Comment (allamand) [reply plugins/kubectl-casskop]: ok :)

**Pull request merged**